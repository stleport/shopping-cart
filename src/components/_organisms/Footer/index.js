import React from 'react';

const Footer = () => {
  return (
    <footer className="App__footer">
      <a
        className="App__link"
        href="https://frichti.co"
        target="_blank"
        rel="noopener noreferrer"
      >
        frichti.co
      </a>
    </footer>
  );
};

export default Footer;