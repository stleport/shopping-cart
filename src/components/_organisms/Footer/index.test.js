import React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';
import Footer from '.';

function FooterSetup() {
  const props = {
    description: 'Description',
    quantity: 1
  };

  const FooterJSX = (
    <Footer {...props} />
  );

  const FooterComponent = shallow(FooterJSX);
  return {
    FooterComponent,
    FooterJSX,
    props
  };
}

describe('Footer', () => {

  it('renders correctly Footer component', () => {
    const { FooterJSX } = FooterSetup();
    const FooterComponent = renderer.create(FooterJSX).toJSON();
    expect(FooterComponent).toMatchSnapshot();
  });
});
