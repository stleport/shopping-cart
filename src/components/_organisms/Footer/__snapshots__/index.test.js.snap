// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders correctly Footer component 1`] = `
<footer
  className="App__footer"
>
  <a
    className="App__link"
    href="https://frichti.co"
    rel="noopener noreferrer"
    target="_blank"
  >
    frichti.co
  </a>
</footer>
`;
